### PR TITLE
fix(export): Filter export for pre required filters

### PR DESCRIPTION
**Pull request by viveckadhriyaa**

### CODE REVIEW

File: lib/spotlight_search/exportable_columns_v2.rb [hunk 1]
```diff
@@ -32,6 +32,10 @@ def export_columns(*record_fields, **associated_fields)
         self.enabled_columns = [*record_fields, **associated_fields]
       end
 
+      def default_scopes_for_export(*filter_scopes)
```
Comment (sajinmp) [inline lib/spotlight_search/exportable_columns_v2.rb:35]: If an array is needed, `filter_scopes` is enough bro. [*filter_scopes] and filter_scopes are same if it's an array.

File: lib/spotlight_search/exportable_columns_v2.rb [hunk 2]
```diff
@@ -32,6 +32,10 @@ def export_columns(*record_fields, **associated_fields)
...
+        self.default_filters = [*filter_scopes]
```
Comment (sajinmp) [inline lib/spotlight_search/exportable_columns_v2.rb:36]: If an array is needed, filter_scopes is enough bro. [*filter_scopes] and filter_scopes are same if it's an array.

**Review by sajinmp — CHANGES_REQUESTED** [review]
One change bro

**Comment by camiloforero** [comment]
I agree with Sajimp's change. And other than that everything looks straightforward and idiomatic. If it works in practice, codewise it's good to go

**Pull request merged**